subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -78,7 +78,7 @@ The dependencies for annotation processors are not added to an IDEA module's cla
 
 ==== Updates to bundled Gradle dependencies
 
-- Kotlin has been updated to https://blog.jetbrains.com/kotlin/2020/03/kotlin-1-3-70-released/[Kotlin 1.3.70].
+- Kotlin has been updated to https://github.com/JetBrains/kotlin/releases/tag/v1.3.71[Kotlin 1.3.70].
```

Comment:
:x: Gradle 6.3 was released with 1.3.70. This should stay unchanched. Instead a section `Updates to bundled Gradle dependencies` with the upgrade to 1.3.71 should be added in `Upgrading from 6.3 above.

Reply:
Done. Thanks!